client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -201,6 +238,44 @@ describe('FeedbackGeneratorService', function() {
         'ZeroDivisionError: integer division or modulo by zero on line 5');
     });
 
+    it('should throw an error if the line number index is less than 0',
+        function() {
```

Comment:
Done! I agree that consistency is important. I think this one just slipped my attention.